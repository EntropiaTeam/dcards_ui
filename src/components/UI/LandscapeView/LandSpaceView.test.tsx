import React from 'react';
import renderer from 'react-test-renderer';
import LandSpaceView from '.';

interface IntlMock {
  useTranslation: () => void;
}

interface TranslationMock {
  t: () => void;
}

jest.mock('react-i18next', (): IntlMock => ({
  useTranslation: (): TranslationMock => ({
    t: (key: string): string => key
  })
}));

describe('<LandSpaceView />', () => {
  const LandSpaceViewComponent = renderer.create(<LandSpaceView />).toJSON();

  it('should render LandSpaceView component correctly', () => {
    expect(LandSpaceViewComponent).toMatchSnapshot();
  });
});
