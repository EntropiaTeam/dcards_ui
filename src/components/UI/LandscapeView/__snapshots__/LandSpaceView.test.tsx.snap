// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LandSpaceView /> should render LandSpaceView component correctly 1`] = `
<div
  className="MuiBox-root MuiBox-root-6 makeStyles-root-1"
>
  <div
    className="MuiBox-root MuiBox-root-7 makeStyles-content-2"
  >
    <svg
      fill="none"
      height="106"
      style={
        Object {
          "height": 158,
          "width": 150,
        }
      }
      viewBox="0 0 112 106"
      width="112"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M20.0479 35.7966C16.1631 30.7105 21.3619 25.9847 24.4469 24.2576L49.0829 5.4409C52.5796 2.77014 56.0841 5.5462 57.3992 7.26807L105.048 69.6528C108.204 73.7852 106.079 77.044 104.622 78.1568L76.4103 99.705C72.5957 102.619 69.0118 99.9032 67.6966 98.1813L20.0479 35.7966Z"
        fill="white"
        stroke="black"
      />
      <path
        d="M19.6228 26.0548C16.5174 28.4267 15.9183 32.8612 18.2848 35.9596L67.133 99.9146C69.4995 103.013 73.9354 103.602 77.0408 101.23L105.53 79.4699C108.636 77.098 109.235 72.6635 106.868 69.5651L58.0203 5.61009C55.6538 2.51172 51.2179 1.92282 48.1124 4.29474L19.6228 26.0548ZM29.1467 26.9481C29.787 27.2605 30.5487 27.1848 31.1149 26.7523L45.7615 15.5654C46.3277 15.1329 46.6012 14.418 46.4683 13.7181L45.7029 9.68826L49.8264 6.53878C51.6897 5.11563 54.3512 5.46897 55.7711 7.32799L104.619 71.283C106.039 73.142 105.68 75.8027 103.816 77.2259L75.3269 98.9859C73.4636 100.409 70.802 100.056 69.3821 98.1967L20.534 34.2417C19.1141 32.3827 19.4735 29.722 21.3368 28.2988L25.4603 25.1493L29.1467 26.9481Z"
        fill="#A0A0A0"
      />
      <path
        d="M11.4201 67.8275C5.23815 66.1711 6.47033 59.2543 7.85917 56.003L15.8826 26.0593C17.0214 21.8092 21.4917 21.8683 23.5846 22.429L99.4098 42.7463C104.433 44.0922 104.739 47.9704 104.265 49.7412L95.0767 84.0316C93.8344 88.668 89.3381 88.7056 87.2453 88.1448L11.4201 67.8275Z"
        fill="white"
        stroke="black"
      />
      <path
        d="M5.13574 60.3716C4.12436 64.1461 6.35733 68.024 10.1232 69.033L87.8571 89.8618C91.623 90.8708 95.4957 88.629 96.5071 84.8545L105.786 50.2269C106.797 46.4524 104.564 42.5745 100.798 41.5655L23.0642 20.7367C19.2983 19.7277 15.4255 21.9695 14.4142 25.744L5.13574 60.3716ZM13.224 55.2645C13.9218 55.121 14.4789 54.5959 14.6632 53.9078L19.4333 36.1056C19.6177 35.4174 19.3979 34.6842 18.8653 34.211L15.7987 31.4868L17.1417 26.4749C17.7485 24.2102 20.0721 22.8651 22.3317 23.4705L100.066 44.2992C102.325 44.9047 103.665 47.2314 103.058 49.4961L93.7796 84.1236C93.1728 86.3883 90.8491 87.7334 88.5896 87.128L10.8557 66.2993C8.5962 65.6938 7.25641 63.3671 7.86324 61.1024L9.20617 56.0905L13.224 55.2645Z"
        fill="#C8C8C8"
      />
      <path
        d="M7.99998 101.5C1.59997 101.5 0.999978 94.5 1.49998 91L1.49998 60C1.49998 55.6 5.83331 54.5 7.99998 54.5L86.5 54.5C91.7 54.5 93 58.1667 93 60L93 95.5C93 100.3 88.6667 101.5 86.5 101.5L7.99998 101.5Z"
        fill="white"
        stroke="black"
      />
      <path
        d="M1.87629e-06 95.9245C2.0471e-06 99.8322 3.16056 103 7.0593 103L87.5353 103C91.434 103 94.5946 99.8322 94.5946 95.9245L94.5946 60.0755C94.5946 56.1678 91.434 53 87.5353 53L7.0593 53C3.16056 53 1.38469e-07 56.1678 3.09279e-07 60.0755L1.87629e-06 95.9245ZM6.49085 88.8981C7.12776 88.5789 7.52992 87.9275 7.52992 87.2151L7.52992 68.7849C7.52992 68.0725 7.12776 67.4211 6.49085 67.1019L2.82372 65.2642L2.82372 60.0755C2.82372 57.7309 4.72005 55.8302 7.0593 55.8302L87.5353 55.8302C89.8745 55.8302 91.7709 57.7309 91.7709 60.0755L91.7709 95.9245C91.7709 98.2691 89.8745 100.17 87.5353 100.17L7.0593 100.17C4.72005 100.17 2.82372 98.2691 2.82372 95.9245L2.82372 90.7358L6.49085 88.8981Z"
        fill="#E8E8E8"
      />
      <path
        d="M107.839 7.99986C107.839 1.59985 100.839 0.999856 97.3391 1.49986L66.3391 1.49986C61.9391 1.49986 60.8391 5.83319 60.8391 7.99986L60.8391 86.4999C60.8391 91.6999 64.5058 92.9999 66.3391 92.9999L101.839 92.9999C106.639 92.9999 107.839 88.6665 107.839 86.4999L107.839 7.99986Z"
        fill="white"
        stroke="black"
      />
      <path
        d="M102.264 0C106.171 0 109.339 3.16056 109.339 7.0593V87.5353C109.339 91.434 106.171 94.5946 102.264 94.5946H66.4146C62.5069 94.5946 59.3391 91.434 59.3391 87.5353V7.0593C59.3391 3.16056 62.5069 0 66.4146 0H102.264ZM95.2372 6.49085C94.918 7.12776 94.2666 7.52992 93.5542 7.52992H75.124C74.4116 7.52992 73.7602 7.12776 73.441 6.49085L71.6033 2.82372H66.4146C64.07 2.82372 62.1693 4.72005 62.1693 7.0593V87.5353C62.1693 89.8745 64.07 91.7709 66.4146 91.7709H102.264C104.608 91.7709 106.509 89.8745 106.509 87.5353V7.0593C106.509 4.72005 104.608 2.82372 102.264 2.82372H97.075L95.2372 6.49085Z"
        fill="black"
      />
      <path
        d="M20.0593 92C19.4062 82.6667 23.88 64 47 64"
        stroke="#C8102E"
        strokeWidth="2"
      />
      <path
        d="M43 60L47 64L43 68"
        stroke="#C8102E"
        strokeWidth="2"
      />
      <path
        d="M76 49.5L82.5 56L94.5 44"
        stroke="#C8102E"
        strokeWidth="4"
      />
    </svg>
    <div
      className="MuiBox-root MuiBox-root-8 makeStyles-textBlock-3"
    >
      <h3
        className="MuiTypography-root makeStyles-title-4 MuiTypography-h3"
      >
        pages.dummyTitle
      </h3>
      <p
        className="MuiTypography-root makeStyles-text-5 MuiTypography-body1"
      >
        pages.dummyText
      </p>
    </div>
  </div>
</div>
`;
